From 810406e4d8d62327ef24f412bdb1eb6d6eeaa16c Mon Sep 17 00:00:00 2001
From: Denis Lebedev <d2.lebedev@gmail.com>
Date: Sat, 23 Mar 2013 14:42:49 +0300
Subject: [PATCH] Apply iOS changes

---
 ext/psych/yaml/api.c |    2 +-
 gc.h                 |   10 +++++-----
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/ext/psych/yaml/api.c b/ext/psych/yaml/api.c
index 0c4732e..fa437a1 100644
--- a/ext/psych/yaml/api.c
+++ b/ext/psych/yaml/api.c
@@ -1,6 +1,6 @@
 
 #include "yaml_private.h"
-
+#include "config.h"
 /*
  * Get the library version.
  */
diff --git a/gc.h b/gc.h
index 10f634e..611e25a 100644
--- a/gc.h
+++ b/gc.h
@@ -74,13 +74,13 @@ int ruby_get_stack_grow_direction(volatile VALUE *addr);
 # define STACK_UPPER(x, a, b) (stack_growup_p(x) ? (a) : (b))
 #endif
 
-#if STACK_GROW_DIRECTION
+//#if STACK_GROW_DIRECTION
 #define STACK_GROW_DIR_DETECTION
 #define STACK_DIR_UPPER(a,b) STACK_UPPER(0, (a), (b))
-#else
-#define STACK_GROW_DIR_DETECTION VALUE stack_grow_dir_detection
-#define STACK_DIR_UPPER(a,b) STACK_UPPER(&stack_grow_dir_detection, (a), (b))
-#endif
+//#else
+//#define STACK_GROW_DIR_DETECTION VALUE stack_grow_dir_detection
+//#define STACK_DIR_UPPER(a,b) STACK_UPPER(&stack_grow_dir_detection, (a), (b))
+//#endif
 #define IS_STACK_DIR_UPPER() STACK_DIR_UPPER(1,0)
 
 #if defined __GNUC__ && __GNUC__ >= 4
-- 
1.7.10.2 (Apple Git-33)

